src/components/Inventory/InventoryContainer.scss

```diff
@@ -0,0 +1,98 @@
+@use "../../styles/partials/variables" as *;
+@use "../../styles/partials/mixins" as *;
+
+.card-container {
+  width: 336px;
+  margin: auto;
+  box-shadow: 0px 2px 5px 0px #13182c1a;
+  background: $color-white;
+  position: relative;
+  top: -70px;
+  border-radius: 10px;
+  padding-top: 10px;
+  padding-left: 30px;
+  padding-right: 30px;
+  padding: 10px 30px 0 30px;
```

Comment:
You dont need line 12-14 since you are using shorthand for `padding` on line 15.